ledger/acctupdates_test.go

```diff
@@ -1046,11 +1046,11 @@ func TestIsWritingCatchpointFile(t *testing.T) {
 
 	au := &accountUpdates{}
 
-	au.catchpointWriting = make(chan struct{}, 1)
+	au.catchpointWriting = -1
```

Comment:
Why not use the same statement used in the production code for setting the value?

Reply:
you mean the `atomic.StoreInt32` ? There is no concurrency here, so I'm not sure why that would make a difference. Keep in mind that it's literally just store a value, nothing else; I think that in the context of this test function isn't not really important. If it was used in a scenario where we try to modify it while there is some go-routine running, than yes - it would be a good idea.